diagnostic.md

```diff
@@ -5,74 +5,81 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The purpose is to do all the behind the scnenes work so that the user can have a good experience.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+the model layer
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+the controller layer
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+I dont rememeber.
```

Comment:
`Because we build our own Client 'views' with HTML/Handlebars/Javascript etcetera.`